Collections/Collections.pier

```diff
@@ -119,8 +122,9 @@ same requests. Let us briefly survey some of the key differences:
 *Some collection classes categorized by implementation technique.>file://figures/CollectionsByImpl|label=collsByImpl*
 
 These categorizations by functionality are not our only concern; we must also
-consider how the collection classes are implemented. As shown in Figure
-*fig:collsByImpl*, five main implementation techniques are employed.
+consider how the collection classes are implemented. As shown in Figure 1.1
```

Comment:
I'm fairly sure it's not Figure 1.1. (The numbering scheme has something to do with chapter number and figure number). I suspect you compiled just the chapter, and LaTeX that generated the PDF gave it number 1.1 (because it thought this was the only chapter).
Anyhow, don't use that number. The `*fig:collsByImpl*` part is supposed to resolve to the right number, if everything is working.